@memZono/plot.m

```diff
@@ -1,20 +1,42 @@
 %% Plot Function for memZono
```

Comment:
From @jruths:
 - [ ] I think we need to require dims to be passed in, otherwise we don't know what order to plot it in.  
 - [ ] The axis labeling is a nice convenience... but I could imagine scenarios where you are plotting multiple memZonos and their dim labels are not matching, even if they are plotted in the same space...

Reply:
1. that was the original implementation... I think it is possible to do either but the motivation w/ this approach was to make checks easier (although idk by how much)

2. This happened in SLAM example... so I agree to eliminate

Reply:
I think we should go ahead and remove plot since it no longer does any substantial work... just call abstract zono plot.